### PR TITLE
Refactor launch app AWS integration

### PR DESCRIPTION
**Pull request by amirfefer**
### Description

Currently, our AWS integration wizard guides users through creating a policy and role resources, attaching them manually via the AWS CLI, and entering the ARN.

By leveraging an AWS CloudFormation stack template, we can streamline this process. Users will be able to achieve this with a simple redirection to AWS console with a dedicated template, eliminating the need for CLI commands or copy any JSON file. The ARN will be retrieved automatically, simplifying the integration with our service.

[HMS-4041](https://issues.redhat.com/browse/HMS-4041)

---

#### Before:

https://github.com/RedHatInsights/sources-ui/assets/11807069/c8589562-2dd4-4b12-a2be-8198f97bee66

#### After:

https://github.com/RedHatInsights/sources-ui/assets/11807069/7b0b2ce4-6c55-486c-b9d8-efd15ca1d069

---

### Checklist ☑️
- [x] PR only fixes one issue or story 
- [ ] Change reviewed for extraneous code 
- [ ] UI best practices adhered to 
- [ ] Commits squashed and meaningfully named 
- [ ] All PR checks pass locally (build, lint, test, E2E)

##
- [ ] _(Optional) QE: Needs QE attention (OUIA changed, perceived impact to tests, no test coverage)_
- [ ] _(Optional) QE: Has been mentioned_
- [ ] _(Optional) UX: Needs UX attention (end user UX modified, missing designs)_
- [x] _(Optional) UX: Has been mentioned_
##

### CODE REVIEW

**Comment by fhlavac** [comment]
@amirfefer it looks like the tests need to be updated. Also, did anyone from UX review the changes?

I noticed especially the button not being in sync with what we have for Azure:
Snímek obrazovky 2024-06-28 v 12 00 51
vs
Snímek obrazovky 2024-06-28 v 12 02 29

And also the tooltip looks a bit weird
Snímek obrazovky 2024-06-28 v 12 01 35

File: src/components/addSourceWizard/hardcodedComponents/aws/provisioningArn.js [hunk 1]
```diff
@@ -143,85 +95,129 @@ export const IAMRoleDescription = () => {
...
+                  defaultMessage="Return to this window and click {next} to retrieve the new Role's ARN."
+                  values={{ next: <strong>Next</strong> }}
+                />
+              </TextListItem>
+            </TextList>
+          </TextContent>
+        </StackItem>
+        <StackItem>
+          <Bullseye>
+            <Button isDisabled={!provAwsAccount} isLoading={!provAwsAccount} onClick={openAWSStackPage} variant="primary">
```
Comment (fhlavac) [inline src/components/addSourceWizard/hardcodedComponents/aws/provisioningArn.js:177]: Here we could directly use 
```
<Button
        component="a"
        target="_blank"
        href={link}
        ...
```

**Comment by maryshak1996** [comment]
Hey there @fhlavac asked me to do a quick UX review here -- overall looks good, but I've got a few suggestions:

- We don't have to capitalize the 'r' in 'roles' every time we say 'IAM Role' -- we can just say 'IAM role'
- For actions that a user clicks on we should use **'[words]'** as well as bold the text. For example, instead of "Click on **Connect AWS** to open AWS...", we should say "Click on **'Connect AWS'** to open AWS..."
- Make the button will width to match the lighthouse example
- Instead of having a very large tooltip that expands beyond the size of the modal itself, we could do an expandable section that says 'See role permissions'. It would be closed by default (like you see in the first screenshot below) but then could be triggered open if the user would like (second screenshot).
image
image

Hope this helps!

**Comment by amirfefer** [comment]
Thanks @fhlavac and @maryshak1996!
I've addressed your comments

Screenshot 2024-07-02 at 21 02 33

**Review by fhlavac — APPROVED** [review]

**Comment by fhlavac** [comment]
This should fix the build, just we need to bump the FEC version when it is released https://github.com/RedHatInsights/frontend-components/pull/2028

**Comment by fhlavac** [comment]
[@redhat-cloud-services/tsc-transform-imports/v/1.0.13](https://www.npmjs.com/package/@redhat-cloud-services/tsc-transform-imports/v/1.0.13)

**Comment by ezr-ondrej** [comment]
@fhlavac should we bump the version in this PR?

**Comment by fhlavac** [comment]
@ezr-ondrej yes, please

**Comment by ezr-ondrej** [comment]
outch: `Error: building at STEP "COPY LICENSE /licenses/": checking on sources under "/var/lib/jenkins/workspace/RedHatInsights-sources-ui-pr-check/build/container_workspace": copier: stat: "/LICENSE": no such file or directory`

**Comment by ezr-ondrej** [comment]
Trying to fix it here: https://github.com/RedHatInsights/sources-ui/pull/1216

**Comment by ezr-ondrej** [comment]
@RedHatInsights/platform-experience this would be amazing to have if you get to it :pray:

**Review by fhlavac — APPROVED** [review]

**Pull request merged**